dapp/src/components/DocsDrawer.tsx

```diff
@@ -1,25 +0,0 @@
-import React from "react"
-import {
-  Drawer,
-  DrawerBody,
-  DrawerContent,
-  DrawerOverlay,
-} from "@chakra-ui/react"
-import { useDocsDrawer } from "#/hooks"
-import { TextMd } from "#/components/shared/Typography"
-
-export default function DocsDrawer() {
-  const { isOpen, onClose } = useDocsDrawer()
-
-  return (
-    <Drawer size="xl" placement="right" isOpen={isOpen} onClose={onClose}>
-      <DrawerOverlay mt="header_height" />
```

Comment:
I think we can already remove `header_height` because we won't need it anymore.

Reply:
Ref commit: a700cd4